core/src/main/scala/chisel3/BlackBox.scala

```diff
@@ -27,6 +27,7 @@ package experimental {
   case class IntParam(value: BigInt) extends Param
   case class DoubleParam(value: Double) extends Param
   case class StringParam(value: String) extends Param
+  case class PrintableParam(value: chisel3.Printable, id: BaseModule) extends Param
```

Comment:
```suggestion
  /** Creates a parameter from the Printable's resulting format String */
  case class PrintableParam(value: chisel3.Printable, context: BaseModule) extends Param
```
Just some minor nits to make what this does a little more clear.